.github/actions/run-integration-tests/action.yml

```diff
@@ -45,10 +23,11 @@ runs:
               echo "IntegrationTestsJob: Just wait until the clean-up finishes..."; \
               return; \
             fi; \
-            echo "IntegrationTestsJob: Notifying the integration tests container about the cancellation"; \
-            docker kill --signal="SIGINT" ig-integration-tests > /dev/null; \
-            echo "IntegrationTestsJob: Waiting for the integration tests container to finish"; \
-            docker wait ig-integration-tests & wait $!; \
+            echo "IntegrationTestsJob: Notifying the integration tests process about the cancellation"; \
+            PIDS=$(jobs -p)
```

Comment:
Did you try this using the cancellation?

Reply:
Yes, just tested it [here](https://github.com/mqasimsarfraz/inspektor-gadget/actions/runs/3090481174/jobs/5000096159). Thanks for suggesting I was missing a `;` which would have broken the cleanup. :smile:

Reply:
I don't see any reason why it shouldn't work. `;\` is bit error prone as well. I will try without it.

Reply:
I remember it didn't work, but I didn't investigate it too much 🙈